cookbooks/tian_parameterization_kinematic_slab/coupled_two_phase_tian_parameterization_kinematic_slab.prm

```diff
@@ -0,0 +1,234 @@
+# This is a cookbook which simulates a portion of 2D subduction system with
+# a hydrated sediment layer, MORB layer, gabbro layer, and peridotite
+# mantle wedge. The layers dehydrate as they are subducted, with the solid-fluid
+# reactions governed by an approximation published by Tian et al., 2019.
+# The model simulates the upper 20 km of a slab subducting with a dip of 45 degrees
+# and a convergence rate of 5 cm/yr. The Cartesian box is rotated such that the
+# x-axis aligns with the slab surface, and the y-axis is orthogonal to the slab
+# surface, while gravity is imposed at an angle of 45 degrees. A surface pressure
+# of 1 GPa is imposed to account for the model simulating a portion of the slab
+# that is already subducted.
+# For stability, the slab first subducts for 100 kyr before the hydration
+# is added, and the slab surface temperature diffuses to the mantle over 10 km
+# perpendicular to the slab surface. From slab surface to the base of the model,
+# the composition is comprised of a 5km thick layer of sediment with 3 wt% water,
+# below is a 7km thick layer of MORB with 2 wt% water, and below that is an
+# 8 km thick layer of gabbro with 1 wt% water.
+
```

Comment:
I would also add some information about what type of equations you are solving - here it is an incompressible formulation with fully coupled two-phase flow. This would also be a good place to cite relevant papers (Juliane and Timo's melting papers, papers from Cian Wilson, Ikudo Wada/Nestor Cerpa, and Tian/Katz/May).